dwave/cloud/client.py

```diff
@@ -81,60 +82,86 @@
+    hybrid :term:`sampler` classes.
 
-    Manages workers and handles thread pools for submitting problems, cancelling tasks,
-    polling problem status, and retrieving results.
+    Manages workers and handles thread pools for submitting problems, cancelling
+    tasks, polling problem status, and retrieving results.
 
     Args:
-        endpoint (str):
+        endpoint (str, optional):
             D-Wave API endpoint URL.
 
         token (str):
             Authentication token for the D-Wave API.
 
-        solver (dict/str):
-            Default solver features (or simply solver name).
+        solver (dict/str, optional):
+            Default solver features (or simply solver name) to use in
+            :meth:`~dwave.cloud.client.Client.get_solver`.
+
+            Defined via dictionary of solver feature constraints
+            (see :meth:`~dwave.cloud.client.Client.get_solvers`).
+            For backward compatibility, a solver name, as a string, is also
+            accepted and converted to ``{"name": <solver name>}``.
 
-        proxy (str):
+            If undefined, :meth:`~dwave.cloud.client.Client.get_solver` uses a
+            solver definition from environment variables, a configuration file,
+            or falls back to the first available online solver.
+
+        proxy (str, optional):
             Proxy URL to be used for accessing the D-Wave API.
 
         permissive_ssl (bool, default=False):
             Disables SSL verification.
 
         request_timeout (float, default=60):
-            Connect and read timeout (in seconds) for all requests to the D-Wave API.
+            Connect and read timeout (in seconds) for all requests to the
+            D-Wave API.
 
-        polling_timeout (float, default=None):
-            Problem status polling timeout (in seconds), after which polling is aborted.
+        polling_timeout (float, optional):
+            Problem status polling timeout (in seconds), after which polling is
+            aborted.
 
         connection_close (bool, default=False):
-            Force HTTP(S) connection close after each request.
+            Force HTTP(S) connection close after each request. Use to prevent
+            intermediate network equipment closing idle connections.
+
+        headers (dict/str, optional):
+            Newline-separated additional HTTP headers to include with each
+            API request, or a dictionary of (key, value) pairs.
+
+        client_cert (str, optional):
+            Client certificate file path.
+
+        client_cert_key (str, optional):
+            Client certificate key file path.
+
+        poll_backoff_min (float, default=0.05):
+            Problem status polling exponential back-off base period, in seconds.
 
-        headers (dict/str):
-            Additional HTTP headers.
+        poll_backoff_max (float, default=60):
+            Problem status polling exponential back-off max period, in seconds.
```

Comment:
Okay, compared to a row of seven nouns, five wasn't too bad. "Maximum exponential back-off period, in seconds, for polling problem status."